data/json/monsters/insect_spider.json

```
@@ -1896,6 +1651,9 @@
     "dodge": 8,
     "armor_bash": 2,
     "armor_cut": 5,
+    "weakpoint_sets": [ "wps_arthropod", "wps_arthropod_flying", "wps_arthropod_hymenoptera" ],
+    "families": [ "prof_intro_biology", "prof_wp_basic_bug", "prof_wp_hymenoptera" ],
+    "weakpoints": [ { "id": "stinger", "coverage": 0 } ],
```

If you just want to remove the `stinger` weakpoint here, it might be better to use:
```suggestion
    "delete": { "weakpoints": [ { "id": "stinger" } ] },
```

At least, I would suggest that if deleting weakpoints wasn't broken at the moment :laughing:
I have a fix ready in #58769, and I think once that is merged you should use the `"delete"` definition instead

(Same for other places where coverage is set to 0)

---

Nice! Yeah, I have a tendency to distrust extend/delete but it will be the way to go here.